x-pack/elastic-agent/_meta/config/common.p2.yml.tmpl

```diff
@@ -23,13 +23,13 @@ datasources:
           - metricset: filesystem
             dataset: system.filesystem
 
-settings.monitoring:
-  # enabled turns on monitoring of running processes
-  enabled: true
-  # enables log monitoring
-  logs: true
-  # enables metrics monitoring
-  metrics: true
+# settings.monitoring:
```

Comment:
In Beats we tried not to have a space after `#` for the things that are actual settings. Like this commenting out is easier and it is more obvious what is a setting and what a description.

Also below we have for the comments two `#` which I think makes it less readable.

Reply:
will handle in separate PR